### ISSUE TITLE
is BatchNorm binarized?

### ISSUE THREAD

**Issue by emiliopaolini** [larq/larq#714]
Hi everyone, i have a simple question: if i look at the summary of every model in larq, it looks like that computation for batch normalization has been binarized. I would like to know how this is done. If it adopts the same strategy as convolution/dense layer or if i am wrong and batch normalization is not binarized at all. Thank you very much!

**Comment by emiliopaolini** [comment]
So, in order to understand: can i apply the same equations that are applied to the non-quantized case? And then inference is still performed using binary fused parameters right?
